aiotg/bot.py

```diff
@@ -100,6 +101,34 @@ def run(self):
         except KeyboardInterrupt:
             self.stop()
 
+    def run_with_reloader(self, **kwargs):
```

Comment:
Maybe we should keep this function "private" but expose reloading through debug (or reload) option to run(), like flask does?